README.md

```diff
@@ -1,96 +1,57 @@
-
-### How to get help
-![image](/steps-to-ask-for-help.png)
-
-## Folder Structure
-
-### Project 1
-```markdown
-|-- data
-|---- processed
-|---- raw
-|---- sql
-|-- reports
-|-- src
-|-- README.md
-|-- .gitignore
-```
-
-### Project 2
-```markdown
-|-- data
-|---- processed
-|---- raw
-|---- sql
-|-- experiments
-|-- models
-|-- reports
-|-- src
-|-- README.md
-|-- .gitignore
-```
-
-* **Data:** Contains the raw, processed and final data. For any data living in a database, make sure to export the tables out into the `sql` folder, so it can be used by anyone else.
-* **Experiments:** A folder for experiments
-* **Models:** A folder containing trained models or model predictions
-* **Reports:** Generated HTML, PDF etc. of your report
-* **src:** Project source code
-* README: This file!
-* .gitignore: Files to exclude from this folder, specified by the Technical Facilitator
+## README TEAM #3
 
+#### VITALII / TJ / ADITHYA / AMRIT
+
+How will you select your dataset?
+We selected our dataset from Kaggle, specifically the "Credit Score Classification" dataset by Paris Rohan. This dataset was chosen due to its relevance to our analysis objectives, its comprehensive features, and the fact that it provides sufficient data points to perform meaningful statistical analysis and visualizations.
+
+How will you make sure all team members can contribute to the project?
+To ensure all team members can contribute to the project, we will:
+
+We used version control system like Git to manage our codebase, allowing everyone to work on different parts of the project simultaneously.
+Assign specific roles and tasks based on each team member's strengths and interests, such as data cleaning, visualization, model development, and documentation.
+Conduct regular team meetings to discuss progress, address any issues, and plan the next steps.
+
+How will you make decisions?
+Decisions will be made collaboratively through group discussions and consensus.
+
+For more complex or technical decisions, we will:
+
+Present and discuss the options during team meetings.
+Use data and empirical evidence to support our choices.
+If necessary, vote on the best course of action.
+Ensure all voices are heard and considered to maintain a democratic process.
+
+What is the question you're trying to answer through your data analysis?
+The primary question we are trying to answer through our data analysis is: "What is the relationship between income and invested funds in the context of credit scores?" By exploring this relationship, we aim to understand how financial behavior impacts creditworthiness.
+
+What tasks need to be completed to get to your final output?
+To achieve our final output, we need to complete the following tasks:
+
+Data Preprocessing:
```

Comment:
Maybe explain a little bit more about how you did this? You're currently explaining what you should do, but I need you to explain what you ended up doing.